portfolio/src/main/webapp/index.html

```diff
@@ -27,21 +27,25 @@ <h1 style="text-align:center"> Sean Tisdale Portfolio </h1>
                 computer programmming intereted me the most. After completing the course I did research and saw all of the things someone could do with computer science which always made me interested 
                 because sticking to one thing forever was always boring to me.I am currently mostly interested in databases and AI in computer science. hobbies I have are playing games, watching anime or tv,
                 and eating. I am currently trying to get into cooking more but it's a process so I'll give it time before calling it a hobby. </p>
-
             <br>
             <p>Click here to get a random fact:</p>
             <button onclick="addRandomFact()">Facts</button>
             <div id="fact-container"></div>
             <br>
             <p>Click here to get your comments!:</p>
-            <button onclick="addMessage()">Messages</button>
+            <button onclick="addMessage()"> Show Messages</button>
             <div id="message-container"></div>
             <br>
             <form action="/data" method="POST">
-            <p> Leave a comment!</p><br>
-            <input type="text" name="comments">
-            <input type="submit" value="Submit"/>
+                <p> Leave a comment!</p><br>
+                <input type="text" name="comments">
+                <input type="submit" value="Submit"/>
+                <br>
+                <button onclick="deleteData()">Delete all comments</button>
```

Comment:
Thats because after doing the delete you need to do either of the following:
1. Make the delete servlet return a redirect URL to reload the page.
2. Have the javascript logic do a page reload after calling the delete servlet.

Reply:
You can do that fix in a followup PR. I'll approve this PR for now.